.azure-pipelines/templates/validation-request.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+- task: ManualValidation@0
```

Comment:
according to [the documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/manual-validation?view=azure-devops&tabs=yaml) this cannot be used in an "agent job" (has to use agent-less).
Also the trade off of using this task vs environment + environment policies is we'll loose the ability to track which release is deployed to which environment. https://docs.microsoft.com/en-us/azure/devops/pipelines/process/environments?view=azure-devops

Reply:
As of the most recent commit, this file would not be used. Will remove before final merge.

Reply:
reminder for this file